src/validation/trust.details.validation.ts

```diff
@@ -38,7 +38,7 @@ const setIsTrustToBeCeasedFlagOnBody = () => {
 export const checkTrustStillInvolved = (req: Request): boolean => {
   const appData: ApplicationData = getApplicationData(req.session);
 
-  const isUpdateOrRemove: boolean = appData.entity_number !== undefined;
+  const isUpdateOrRemove: boolean = !!appData.entity_number;
```

Comment:
Is this "!!" widely used? Could we explicitly check whether this is truthy instead?

Reply:
It is already used in a few places within the OE web app, so not introducing something entirely new here.

If using a single ! (to check truthy value) I then need to NOT again on the next line, when checking the value, so after a quick discussion with @markpit we decided this was the best approach.